<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>日志窗口</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QDockWidget" name="dockWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>174</width>
       <height>25</height>
      </size>
     </property>
     <property name="features">
      <set>QDockWidget::NoDockWidgetFeatures</set>
     </property>
     <property name="allowedAreas">
      <set>Qt::AllDockWidgetAreas</set>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>0</y>
         <width>352</width>
         <height>27</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toolButton_Clear">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_Error">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_Debug">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_Info">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>178</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit_log"/>
   </item>
  </layout>
  <action name="actionClearLog">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icos/cBreak.png</normaloff>:/icos/cBreak.png</iconset>
   </property>
   <property name="text">
    <string>ClearLog</string>
   </property>
   <property name="toolTip">
    <string>清理日志</string>
   </property>
  </action>
  <action name="actionDebugLevel">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icos/debug48.png</normaloff>:/icos/debug48.png</iconset>
   </property>
   <property name="text">
    <string>DebugLevel</string>
   </property>
   <property name="toolTip">
    <string>设置为DebugLevel</string>
   </property>
  </action>
  <action name="actionInfoLevel">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icos/preferences-calltips.png</normaloff>:/icos/preferences-calltips.png</iconset>
   </property>
   <property name="text">
    <string>InfoLevel</string>
   </property>
   <property name="toolTip">
    <string>设置为Info等级</string>
   </property>
  </action>
  <action name="actionErrorLevel">
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icos/exceptions.png</normaloff>:/icos/exceptions.png</iconset>
   </property>
   <property name="text">
    <string>ErrorLevel</string>
   </property>
   <property name="toolTip">
    <string>设置为错误级别</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
